README.md

```diff
@@ -1,73 +1,117 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+## hello-rails-react
 
+> Created to learn how React and Rails work together.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- [x] Frontend: [React-Redux](https://react-redux.js.org/)
+- [x] Backend: [Ruby on Rails](https://rubyonrails.org/)
+- [x] Linter: [Rubocop](https://rubocop.org/)
+- [x] Code Editor: [VS Code](https://code.visualstudio.com/)
+- [x] Database: [PostgreSQL](https://www.postgresql.org/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+1- Copy this link:
+```
+$ https://github.com/brahimdidi/hello-rails-react.git
```

Comment:
- [ ] Kindly fix your documentation link here. It should be: `https://github.com/brahimdidi/hello-react-from-rails`

Reply:
done